VideoLocker/res/values/errors.xml

```diff
@@ -10,10 +13,14 @@
     <string name="reset_no_network_title">Connection Error</string>
     <!-- Alert dialog content shown when a network request fails because the user has no internet connection -->
     <string name="reset_no_network_message">You are not connected to the Internet. Please check your Internet connection.</string>
+    <!-- An error ocurred due to some kind of network failure -->
+    <string name="network_connected_error">Connection error. Please check your internet connection and try again.</string>
```

Comment:
The capitalization of "internet" should be consistent between error messages. Also the "and try again bit" should be consistent with the `reset_no_network_message` string.